src/components/books.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Books = () => (
+  <div className="booksContainer">
+    <ul>
+      <li>
+        Atomic Habits
+        <span>James Clear</span>
+      </li>
+      <li>
+        The Spooks secret
+        <span>Joseph Delaney</span>
+      </li>
+      <li>
+        Alice in wonderland
+        <span>Lewis carrol</span>
+      </li>
+      <li>
+        Pride and Prejudice
+        <span>Jane Austen</span>
+      </li>
+    </ul>
+    <form>
+      <input type="text" placeholder="Book title..." />
+      <input type="text" placeholder="Book Author" />
+      <input type="submit" />
+    </form>
+  </div>
+);
```

Comment:
Nice Job done so far, You have created the `books` page and the `categories` page as components👍. But for the books page, I suggest you take note of the following and make the necessary change👍.

- [ ] Just as stated in the requirements for this milestone you are to create a component that;
- Contains individual book state, as well as a button to delete the book (more like a component to display a single book)😉.
- Contains a list to render individual books (more like components that shows all your books)😉.
- Contains a form to create new books (a form that lets the user add a new book)😉.
- Contains navigation to link to the 2 routes you created (This you have already implemented👍).

<img width="595" alt="components" src="https://user-images.githubusercontent.com/64643995/236205831-d0a3c213-72c5-4197-8ff9-e7f436b296f7.png">

kindly create a component for each of these items and import them to your `books` page where they will be used😉. Breaking your page into components helps for reusability and separation of concerns which makes it easy to make update to your app👍.

Reply:
Alright on it right away